src/generator/gen_unicode.ml.inc

```diff
@@ -0,0 +1,135 @@
+
+let target = Sys.argv.(1)
+
+let categories =
+  Hashtbl.create 1024
+
+let labels =
+  Hashtbl.create 1024
+
+let keepers =
+  ["cc"; "cf"; "cn"; "co"; "cs"; "ll"; "lm"; "lo"; "lt"; "lu"; "mc"; "me";
+   "mn"; "nd"; "nl"; "no"; "pc"; "pd"; "pe"; "pf"; "pi"; "po"; "ps"; "sc";
+   "sk"; "sm"; "so"; "zl"; "zp"; "zs"; "alphabetic"; "ascii_hex_digit";
+   "hex_digit"; "id_continue"; "id_start"; "lowercase"; "math";
+   "other_alphabetic"; "other_lowercase"; "other_math"; "other_uppercase";
+   "uppercase"; "white_space"; "xid_continue"; "xid_start"]
```

Comment:
How did you pick this particular set of categories? Are you just using the ones that were hard-coded before? Should we check if there are new ones that would be of use? (Note that we don't have to be too crazy worrying about it; we can obviously change it later by editing your file. But it would be nice to have a comment explaining why these were picked at least.)

Reply:
Handpicked.. I'm happy to add more as needed.

Reply:
So long as it's easy to add new ones by storing this just in the one place.